### PR TITLE
Add verbose logs to some test passes to catch schannel timeout issue

### PR DESCRIPTION
**Pull request by guhetier**
## Description

Handshake/WithHandshakeArgsustomClientCertificateValidation has been failing with a call to schannel taking over 3sec.

This temporarily enables schannel logs to more test passes in the hope to get enough logs to investigate next repro.

## Testing

CI

## Documentation

N/A

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -145,13 +145,13 @@ jobs:
...
           { config: "Debug", plat: "windows", os: "windows-2022", arch: "x64", tls: "schannel", xdp: "-UseXdp", sanitize: "-SanitizeAddress", build: "-Test" },
           { config: "Debug", plat: "windows", os: "windows-2022", arch: "x64", tls: "schannel", xdp: "-UseXdp", qtip: "-UseQtip", sanitize: "-SanitizeAddress", build: "-Test", log: "Full.Verbose" },
-          { config: "Debug", plat: "windows", os: "windows-2022", arch: "x64", tls: "quictls", build: "-Test" },
+          { config: "Debug", plat: "windows", os: "windows-2022", arch: "x64", tls: "quictls", build: "-Test", log: "Full.Verbose" },
           { config: "Debug", plat: "windows", os: "windows-2022", arch: "x64", tls: "quictls", xdp: "-UseXdp", build: "-Test" },
           { config: "Debug", plat: "windows", os: "windows-2022", arch: "x64", tls: "quictls", xdp: "-UseXdp", qtip: "-UseQtip", build: "-Test", log: "Full.Verbose" },
-          { config: "Debug", plat: "windows", os: "windows-2022", arch: "x64", tls: "openssl", build: "-Test" },
+          { config: "Debug", plat: "windows", os: "windows-2022", arch: "x64", tls: "openssl", build: "-Test", log: "Full.Verbose" },
```
Comment (mtfriesen) [inline .github/workflows/test.yml:154]: We seem to be churning logs on and off a lot in CI.

One solution I've seen is to create a logging session per test case and only save/upload sessions for test cases that fail, at least by default. This avoids the massive log file problem while automatically producing artifacts for flaky or one-off failures.

Shall we do something similar in MsQuic?

Comment (guhetier) [reply .github/workflows/test.yml:154]: That may eventually be a solution.
For now, I don't think it deserves the investment yet (these are to investigate long standing corner case issues, not common ones).
